### DISCUSSION TITLE
Un système de pity pour les item random

### DISCUSSION THREAD

**Discussion by Asemerald** [Crownicles/Crownicles#2553] [Nouvelles idées]
Genre plus t'en achète et plus t'as de la merde, + ta chance d'obtenir un légendaire ou mythique augmente et se reset a l'obtention

**Comment by BastLast** [comment]
Tellement d'exploit possibles

**Reply by Asemerald** [reply]
T'achète un item ta chance d'obtenir un item passe de 1 sur 10 000 a 1 sur 9 999, c'est pas des exploit c'est une nouvelle mécanique qui doit bien sur être balance etc mais jvois pas les exploit dont tu parles

**Comment by QQtin3** [comment]
Système classique de n'importe quel gacha, en vrai je trouve pas l'idée déconnante mais faut absolument imposer une limite, genre pas passer la chance d'un mythique de 0.01% à 10% et être presque sûr d'avoir ça, si c'est bien géré ça peut être une bonne feature, juste que pour le moment ça demande pas mal de travail à setup pour la feature que c'est imo.

Cependant ça donnerait un plus à ceux qui drop tout leurs golds dans les random items des shops

**Reply by aureochocob0n** [reply]
C'est vrai, mais c'est pas idiot non plus de prévoir que ça va être dur à coder pour rendre impossible le cheat de cette méthode

**Reply by QQtin3** [reply]
Je n'ai pas prétendu le contraire, après très honnêtement je vois difficilement les complications en partant d'une base saine, le problème c'est que l'obtention des items etc peut se faire de pleins de manières différentes et il est vrai que faudrait être particulièrement minutieux sur l'implémentation de la feature

**Reply by aureochocob0n** [reply]
Exactement
